slides/Combinatorics.html

```diff
@@ -0,0 +1,210 @@
+                    </section>
+                    
+                    <section>
+                        <p>
+                            \[
+                                _{200}P_{3} = \frac{200!}{(200 - 3)!}
+                            \]
+                        </p>
+                        <p>
+                            Plugging this into your favorite calculator or REPL shows that there are 7,880,400 possible ways to worship everyone's favorite boy
+                        </p>
+                    </section>
+                </section>
+                
+                <section>
+                    <section>
+                        <h2>Off the Wall Number Sets </h2>
+                        <p>
+                            Really obscure number sets that always seem to creep up in competitons as the 'easy' solution to a problem. 
+                        </p>
+                    </section>
+                    
+                    <section>
+                        <h2>Catalan Numbers</h2>
+                        <p>
+                            \[
+                                C_{n} = \frac{1}{n + 1} {2n\choose n} = \frac{(2n)!}{(n + 1)!n!} = \prod_{k = 2}^n\frac{n + k}{k} for \ n \ge 0
+                            \]
+                        </p>
+                        <p>
+                            
+                            <pre><code data-trim>
+1
+1
+2
+5
+14
+42
+132
+429
+1430
+4862
+16798
+...
+                            </code></pre>
+                        </p>
+                    </section>
+                    
+                    <section>
+                        <h2>Why Are The Catalan Numbers Important?</h2>
+                        <p>The people who make the problems said so.</p>
+                    </section>                    
+                    
+                    <section>
+                        <h2>An Example Problem That We've Seen At Competiton</h2>
+                        <p>
+                            A Dyck Word is a string consisting of n X's and n Y's such that no initial segment of the string has more Y's than X's. The count of the Dyck words for a string of length 2n is C<small>n</small>.  
+                        </p>
+                        <br />
+                        <p>
+                            XXXYYY     XYXXYY     XYXYXY     XXYYXY     XXYXYY.
+                        </p>
+                    </section>
+                </section>
+                
+                <section>
+                    <section>
+                        <h2>Itertools </h2>
+                        <p>
+                            Or how Python is once again the superior language.  
+                        </p>
+                    </section>
+                    
+                    <section>
+                        <h2>Basics of Itertools</h2>
+						<pre><code class="python" data-trim>
+itertools.combinations("ABCDEFG", 3)
```

Comment:
I'd drop some comments here, just in case someone looks back and isn't sure. A lot of times these things return generators (especially in Python 3) and it's not clear what happened and what you got back.